Dockerfile

```diff
@@ -31,13 +46,23 @@ RUN apt-get --yes update \
   # `unzip` and `zlib1g-dev` are needed for Bazel.
   unzip \
   zlib1g-dev \
+  # Cleanup
   && apt-get clean \
   && rm --recursive --force /var/lib/apt/lists/* \
   # Print version of various installed tools.
   && git --version \
-  && clang-format -version \
   && shellcheck --version
 
+
+RUN curl -fsSL https://download.docker.com/linux/debian/gpg | apt-key add -
```

Comment:
What are all these flags to `curl`? Could you expand them, so we don't have to remind ourselves what's going on?